### PR TITLE
CreateForm

### PR DESCRIPTION
**Pull request by asnganje**
Added contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -165,8 +165,34 @@ <h4>Skills</h4>
...
     </section>
 
+    <form action="https://formspree.io/f/xdobraqv" method="post" class="form1">
+        <input name="input1" type="text" required class="name" placeholder=" Full name" maxlength="30">
+        <div class="div-intermediate1"></div>
+        <input name="input2" type="email" required class="email" placeholder="Email address">
+        <div class="div-intermediate2"></div>
+        <textarea name="text-area" required id="#" cols="30" rows="10" class="info" placeholder=" Enter text here"></textarea>
+        <button class="button4" type="button">Get in touch</button>
+    </form>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required`
- [ ] Good job with the form so far :clap: :clap: 
- [ ] you missed out a bit style on the figma please see screenshot below

`Your Template`
Screenshot from 2022-07-05 10-49-36

`Figma Template`
Screenshot from 2022-07-05 10-48-42

File: index.html [hunk 2]
```diff
@@ -165,8 +165,34 @@ <h4>Skills</h4>
...
+        <h3>
+            I'm always interested in hearing about new projects, so if you'd 
+            like to chat please get in touch.
+        </h3>
     </section>
 
+    <form action="https://formspree.io/f/xdobraqv" method="post" class="form1">
+        <input name="input1" type="text" required class="name" placeholder=" Full name" maxlength="30">
+        <div class="div-intermediate1"></div>
+        <input name="input2" type="email" required class="email" placeholder="Email address">
```
Comment (mwanawabangona) [inline index.html:179]: - [ ] `required`
- [ ] Please kindly add the Message input  maxlength="500" attribute.

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
## Status: Changes Required

 ## Hello there @asnganje 👋🏽

###  You have done an amazing job so far! 👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand:


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional briefcase
* ✅  Solid pull request scroll
* ✅  Good use of flexbox 🧐
* ✅  Good quality commits

## Required Changes
 _**Check the comments under the review.**_

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @asnganje 

Your project is complete! Great job for adding the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- Validations correctly implemented✔️ 
- No linter errors✔️ 
- form submits successfully ✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**